modules/installation-guide/partials/con_understanding-the-checluster-custom-resource.adoc

```diff
@@ -1,35 +1,34 @@
 [id="understanding-the-checluster-custom-resource_{context}"]
 = Understanding the `CheCluster` Custom Resource
 
-A default deployment of {prod-short} consist in the application of a parametrized `CheCluster` Custom Resource by the {prod} Operator.
+A default deployment of {prod-short} consists of a `CheCluster` Custom Resource parameterized by the {prod} Operator.
 
-`CheCluster` Custom Resource:: 
+`CheCluster` Custom Resource is a {kubernetes} object. You can configure it by editing the `CheCluster` Custom Resource YAML file. The file contains sections to configure each component: `auth`, `database`, `server`, `storage`.
```

Comment:
```suggestion
The `CheCluster` Custom Resource is a {kubernetes} object. You can configure it by editing the `CheCluster` Custom Resource YAML file. This file contains sections to configure each component: `auth`, `database`, `server`, `storage`.
```

Which file are we editing here? I wouldn't necessarily refer to a Kubernetes object on the cluster as a file.